garage/tf/algos/npo.py

```diff
@@ -323,18 +344,16 @@ def _build_policy_loss(self, i):
                         surr_clip = lr_clip * advantages * i.valid_var
```

Comment:
it seems that the recurrent case here is never tested. can you add a test which uses recurrent too, so that this line is covered?

Reply:
Added in tests/garage/tf/algos/test_ppo.py.